src/App.js

```diff
@@ -3,14 +3,17 @@ import AddTodoForm from './AddTodoForm';
 import TodoList from './TodoList';
 
 function App() {
-  const [newTodo, setNewTodo] = React.useState('');
+  const [todoList, setTodoList] = React.useState([]);
+
+  function addTodo(newTodo) {
+    setTodoList([...todoList, newTodo]);
```

Comment:
No need to change this at all, but just as a reference to the example in the assignment, this will do the exact same thing. Oh and `prevTodos` is yet another booger, it's just common practice to call it either `prevState` or `prevWhateverStateYoureReferencing`

```suggestion
    setTodoList((prevTodos) => [...prevTodos, newTodo]);
```